.github/actions/spiced-cuda-build/action.yaml

```diff
@@ -0,0 +1,73 @@
+name: "Spiced CUDA Build Action"
+description: "Set up CUDA, verify prerequisites, build and package spiced with models and CUDA support."
+inputs:
+  cuda-version:
+    description: "The CUDA version to install"
+    required: true
+    default: "12.4.1"
+  cuda-compute-capability:
```

Comment:
@Jeadie  - is `cuda-compute-capability` used only for naming? I don't see where we apply it as build param or env configuration